tide-compression/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+authors = [
+    "Tide Developers",
+]
+description = "Compression-related middleware for Tide"
+documentation = "https://docs.rs/tide-compression"
+keywords = ["tide", "web", "async", "middleware", "compression"]
+categories = ["network-programming", "compression", "asynchronous"]
+edition = "2018"
+license = "MIT OR Apache-2.0"
+name = "tide-compression"
+readme = "README.md"
+repository = "https://github.com/rustasync/tide"
+version = "0.1.0"
+
+[dependencies]
+tide = { path = "../tide" }
```

Comment:
We need a `version` field in order to publish this crate. Let's do this after structure revamp!